src/Compilers/Core/MSBuildTask/LocateEditorConfigFiles.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.CodeAnalysis.BuildTasks
+{
+    public sealed class LocateEditorConfigFiles : Task
+    {
+        [Required]
+        public ITaskItem[] InputFiles { get; set; }
```

Comment:
The ManagedCompiler task refers to this input item as Sources.  Does it make sense to keep the names in sync as they represent the same set of input?